main/src/ca/uwaterloo/flix/language/phase/jvm/BackendObjType.scala

```diff
@@ -872,76 +898,80 @@ object BackendObjType {
         thisLoad() ~ GETFIELD(BeginColField) ~ INVOKEVIRTUAL(StringBuilder.AppendInt32Method) ~
         // create the string
         INVOKEVIRTUAL(JavaObject.ToStringMethod) ~ ARETURN()
-    ))
+    }
   }
 
   case object Global extends BackendObjType with Generatable {
     def genByteCode()(implicit flix: Flix): Array[Byte] = {
```

Comment:
In this PR - all code related to code generation is within the `genByteCode: Unit => Array[Byte]` function. Which means that all code in this file could be rewritten in the classfile lib without touching any other code

Reply:
OK; so this might be starting point for @CadeMichael